lib/language.js

```diff
@@ -22,13 +22,14 @@ function init() {
     , { code: 'pl', language: 'Polski' }
     , { code: 'pt', language: 'Português (Brasil)' }
     , { code: 'ro', language: 'Română' }
+    , { code: 'ru', language: 'Русский' }
     , { code: 'sv', language: 'Svenska' }
     , { code: 'fi', language: 'Suomi' }
   ];
 
   var translations = {
     // Server
-    'Listening on port' : {
+    'Listening on port--' : {-
       cs: 'Poslouchám na portu'
       ,es: 'Escuchando en el puerto'
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Expected an assignment or function call and instead saw an expression.](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/2232517981/issues/source?bid=2452379&fileBranchId=3176126#l34)